src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+.calendar {
+  width: 707px;
```

Comment:
Avoid such a hard-coded values.
This is like a magic number, it is not clear why is it set to 707px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)

Please create a separate file for variables